aws/table_aws_wellarchitected_answer.go

```diff
@@ -0,0 +1,317 @@
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedAnswer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_answer",
+		Description: "AWS Well-Architected Answer",
+		Get: &plugin.GetConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "lens_alias", Require: plugin.Required},
+				{Name: "question_id", Require: plugin.Required},
+				{Name: "workload_id", Require: plugin.Required},
+			},
+			Hydrate: getWellArchitectedAnswer,
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listWellArchitectedWorkloads,
+			Hydrate:       listWellArchitectedAnswers,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "lens_alias", Require: plugin.Optional},
+				{Name: "pillar_id", Require: plugin.Optional},
+				{Name: "workload_id", Require: plugin.Optional},
+				{Name: "milestone_number", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(wellarchitectedv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "question_id",
+				Description: "The ID of the question.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Answer.QuestionId"),
+			},
+			{
+				Name:        "lens_alias",
+				Description: "The alias of the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "workload_id",
+				Description: "The ID assigned to the workload.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "helpful_resource_display_text",
+				Description: "The helpful resource text to be displayed.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedAnswer,
+				Transform:   transform.FromField("Answer.HelpfulResourceDisplayText"),
+			},
+			{
+				Name:        "helpful_resource_url",
+				Description: "The helpful resource URL for a question.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedAnswer,
+				Transform:   transform.FromField("Answer.HelpfulResourceUrl"),
+			},
+			{
+				Name:        "improvement_plan_url",
+				Description: "The improvement plan URL for a question. This value is only available if the question has been answered.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedAnswer,
+				Transform:   transform.FromField("Answer.ImprovementPlanUrl"),
+			},
+			{
+				Name:        "is_applicable",
+				Description: "Defines whether this question is applicable to a lens review.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("Answer.IsApplicable"),
+			},
+			{
+				Name:        "lens_arn",
+				Description: "The ARN of the lens.",
```

Comment:
```suggestion
				Description: "The Amazon Resource Name (ARN) of the lens.",
```

Would you kindly provide an update to the description in order to standardize the ARN (Amazon Resource Name) description?